ui/webui/test/helpers/installer.py

```diff
@@ -93,3 +93,4 @@ def check_prerelease_info(self, is_expected=None):
 
     def quit(self):
         self.browser.click("#installation-quit-btn")
+        self.browser.click("#installation-quit-confirm-btn")
```

Comment:
Does this work locally? My instinct would be to wait for the button to become visible before clicking it.

Reply:
Yep, it does click the button, shutting down the VM. But good point, better check the dialog is visible first & only then click.